main.js

```diff
@@ -20,17 +22,40 @@ saveBtn.addEventListener('click', (event) => {
 
 titleInput.addEventListener('keyup', activateSaveBtn)
 bodyInput.addEventListener('keyup', activateSaveBtn)
+showStarredIdeasBtn.addEventListener('click', createStarredIdeas)
+
 // star.addEventListener('click', changeFavoriteStatus)
 
 cardGrid.addEventListener('click', (event) => {
```

Comment:
sometimes in production quality code, you'll having something like _**event**_ be simplified down to just e. In a programming language like go, those conventions are commmon.